### ISSUE TITLE
[BUG] Better player cant seek video: Playback stalled

### ISSUE THREAD

**Issue by anhnt** [jhomlala/betterplayer#448]
**Describe the bug**
Cant seek video which has duration about 10 mins
Error 1: Failed to load video. Playback stalled
Error 2: Audio is still playing in the background no matter you go to other screen

**To Reproduce**
1. Play a HLS video from URL
2. Seek to middle of video
3. Error display and audio continue playing

**Expected behavior**
This isnt happening with stock AVPlayer. It should load and play video at that position

**Screenshots**
![image](https://user-images.githubusercontent.com/735555/115953407-7e7dc500-a515-11eb-9a9b-c23754ea77cc.png)

**Sample code**
```
BetterPlayer.network(
            "http://demo.unified-streaming.com/video/tears-of-steel/tears-of-steel.ism/.m3u8",
            betterPlayerConfiguration: BetterPlayerConfiguration(
                controlsConfiguration: BetterPlayerControlsConfiguration(
                    backgroundColor: Colors.white.withOpacity(0.5),
                    showControlsOnInitialize: false,
                    textColor: Colors.white,
                    iconsColor: Colors.white,
                    progressBarPlayedColor: Colors.white
                ),
                errorBuilder: (context, errorMessage){
                  return Center(
                    child: Container(padding: EdgeInsets.symmetric(vertical: 50), child: Text(errorMessage),),
                  );
                },
                aspectRatio: 16 / 9,
                autoPlay: true,
                looping: true,
                autoDetectFullscreenDeviceOrientation: true,
                fullScreenAspectRatio: 16 / 9,
                autoDispose: true
            ),
          )
```

**Flutter doctor**
[✓] Flutter (Channel stable, 2.0.3, on macOS 11.2.3 20D91 darwin-x64, locale en-AU)
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
[✓] Xcode - develop for iOS and macOS
[✗] Chrome - develop for the web (Cannot find Chrome executable at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome)
    ! Cannot find Chrome. Try setting CHROME_EXECUTABLE to a Chrome executable.
[!] Android Studio (not installed)
[✓] IntelliJ IDEA Ultimate Edition (version 2021.1)
[✓] VS Code (version 1.55.2)
[✓] Connected device (2 available)

**Better Player version**
- Version: 0.0.64

**Smartphone (please complete the following information):**
 - Device: iPhone XS Max
 - OS: iOS 14.4.x

**Comment by jhomlala** [comment]
This data source is very slow. Anyway there's issue on iOS which will be fixed in next release.

**Comment by anhnt** [comment]
I added details to wrong issue somehow, so I repost here

@jhomlala Thank you for your great work on this lib. I have upgraded to latest version 0.0.65 however the bug is still persisted. It has no prob with android devices, only iOS devices when trying to seek. Please see videos here:
iOS: https://www.loom.com/share/964744ef9f4944d38cbb0a1c255c47c1
android: https://www.loom.com/share/e78a4f6486954953baa236d920b086f1
Thank you

**Comment by anhnt** [comment]
PR: https://github.com/jhomlala/betterplayer/pull/470

**Comment by jhomlala** [comment]
Released in 0.0.66.
